_dev/apps/ui/src/components/product-feed/settings/product-selection/type.ts

```diff
@@ -0,0 +1,39 @@
+export interface Features {
+  value: string;
+  id: string;
+}
+
+export type ProductFilterValue = number|string|boolean|null;
+
+export type ProductFilterErrors = {
+  attribute?: string,
+  condition?: string,
+  value?: string,
+}
+
+export type ProductFilterValues = {
+  id: string,
+  value: string
+}
+
+export type ProductFilter = {
+  id: string,
+  attribute?: string,
+  condition?: string,
+  conditionType?: string,
+  value?: ProductFilterValue,
+  values?: string[],
+  errors?: ProductFilterErrors
+}
+
+export type ProductFilterToSend = {
+  attribute: string,
+  condition?: string,
+  value?: ProductFilterValue,
+  values?: string[],
+}
+
```

Comment:
string type is too large for attribute. and no need to repeat the props for ProductFilterToSend 
```typescript
export type ProductFilter = {
  id: string,
  attribute: "price" | "brand" | "category" | "productId" | "outOfStock",
  condition?: string,
  conditionType?: string,
  value?: ProductFilterValue,
  values?: string[],
  errors?: ProductFilterErrors
}

export type ProductFilterToSend = Pick<ProductFilter, "attribute" | "condition" | "value" | "values">;
```